spec/relativetimeformat.html

```diff
@@ -9,15 +9,16 @@
       <h1>InitializeRelativeTimeFormat (relativeTimeFormat, locales, options)</h1>
 
       <p>
-        The abstract operation InitializeRelativeTimeFormat accepts the arguments _relativeTimeFormat_ (which must be an object), _locales_, and _options_. It initializes _relativeTimeFormat_ as a RelativeTimeFormat object. It performas the following steps:
+        The abstract operation InitializeRelativeTimeFormat accepts the arguments _relativeTimeFormat_ (which must be an object), _locales_, and _options_. It initializes _relativeTimeFormat_ as a RelativeTimeFormat object.
       </p>
+      <p>The following steps are taken:</p>
 
       <emu-alg>
         1. If _relativeTimeFormat_ has an [[InitializedIntlObject]] internal slot with value *true*, throw a *TypeError* exception.
         1. Set _relativeTimeFormat_.[[InitializedIntlObject]] to *true*.
         1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
         1. If _options_ is *undefined*, then
-          1. Let _options_ be ObjectCreate(*%ObjectPrototype%*).
+          1. Let _options_ be ObjectCreate(*null*).
```

Comment:
I’m assuming this was an oversight, but in case this was done deliberately: why?

Reply:
yes, seems an oversight. this object is just used internally to extra the value of each configuration.